lib/src/components/NxCollapsibleItems/__tests__/NxCollapsibleItems.test.tsx

```diff
@@ -20,6 +22,10 @@ describe('NxCollapsibleItems', function() {
       },
       getShallowComponent = enzymeUtils.getShallowComponent(NxCollapsibleItems, minimalProps);
 
+  it('is aliased as NxTreeView', function() {
+    expect(NxCollapsibleItems).toEqual(NxTreeView);
```

Comment:
We want to be sure that we're checking reference equality here

```suggestion
    expect(NxCollapsibleItems).toBe(NxTreeView);
```

Reply:
> We want to be sure that we're checking reference equality here

My intent was that you make that change on all of them

Reply:
Ah yes, updated: 4406b5aa79e338ecc25d69d7839077ccc59accf8